### PR TITLE
All features Included for the Portfolio Skeleton

### PR DESCRIPTION
**Pull request by bettercallsully**
- Fixed merge conflicts and added stylesheets and HTML layouts to separate branches 
- Added image assets to separate branches
- Added every feature to it's own respective branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Portfolio | Sully</title>
+</head>
+<body>
+    <header class="mainheader">
+    <img class="mylogo" src="/myassets/MyLogo.png" alt="Portfolio Logo">
+         <img class="hamburger" src="myassets/Union.png" alt="Hamburger Logo">
+        <nav class="toolbar">
+        </nav>
+        <h2></h2>
+    </header>
```
Comment (Dipeshtwis) [inline index.html:18]: - [ ] The content is supposed to aligned with flexbox.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @bettercallsully, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: All linters are passing and commit is also looking good..

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- The UI design has no text and also menu is overflowing.

Screenshot from 2021-06-05 13-27-47


**_Also Check the comments under the review_**


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bettercallsully** [comment]
Thanks for the feedback! ✅👑

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body>
+    <header class="mainheader">
+    <img class="mylogo" src="/myassets/MyLogo.png" alt="Portfolio Logo">
```
[the remark this answers is not in the document]
Comment (bettercallsully) [reply index.html:13]: Do I render my logo to a basic link? Or just wrap the a tags

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+         <img class="hamburger" src="myassets/Union.png" alt="Hamburger Logo">
```
[the remark this answers is not in the document]
Comment (bettercallsully) [reply index.html:14]: Isn't the hamburger menu supposed to be on the right???

**Pull request closed without merging**